src/Microsoft.ML/LearningPipeline.cs

```diff
@@ -49,12 +49,17 @@ public ScorerPipelineStep(Var<IDataView> data, Var<ITransformModel> model)
     public class LearningPipeline : ICollection<ILearningPipelineItem>
     {
         private List<ILearningPipelineItem> Items { get; } = new List<ILearningPipelineItem>();
+        private readonly int? _seed;
+        private readonly int _conc;
 
         /// <summary>
         /// Construct an empty <see cref="LearningPipeline"/> object.
+        /// 
         /// </summary>
-        public LearningPipeline()
+        public LearningPipeline(int? seed=null, int conc=0)
```

Comment:
>seed=null, int conc=0) [](start = 37, length = 22)

Since it's for the tests for now till we settle on a correct solution, maybe consider make this constructor internal in the meanwhile.

So we'd put something like: 
`[assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.TestFramework, ` into an `AssemblyInfo.cs`. (Or if not `TestFramework` whatever other test projects is appropriate.) We'd retain the original constructor.

Reply:
Absolutely, but for now, I just want to run 10-20 builds to make sure problem is gone.

---
In reply to: [192223871](https://github.com/dotnet/machinelearning/pull/277#discussion_r192223871) [](ancestors = 192223871)

Reply:
Hmm all right. Maybe one of those 10 or 20 while we're waiting the minutes we can introduce it, so if we do this we can just sign off.

---
In reply to: [192224636](https://github.com/dotnet/machinelearning/pull/277#discussion_r192224636) [](ancestors = 192224636,192223871)